src/main/antlr4/br/unb/cic/js/miner/JavaScriptParser.g4

```diff
@@ -374,8 +417,7 @@ objectLiteral
     ;
 
 anonymousFunction
-    : functionDeclaration                                                       # FunctionDecl
```

Comment:
an interesting decision, why?

Reply:
These are all updates from https://github.com/antlr/grammars-v4/blob/master/javascript/javascript/JavaScriptParser.g4 . I personally don't know what all changes are dealing with. But, there are bug reports and comments for all of them in the https://github.com/antlr/grammars-v4/ repo